xml/System.Security/CodeAccessPermission.xml

```diff
@@ -971,8 +971,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Throws a <see cref="T:System.Security.SecurityException" /> at run time if the security requirement is not met.</summary>
```

Comment:
Should the exception be added separately as an `<exception cref>`?

Reply:
Definitely.

Reply:
@bartonjs I'm moving the current summary text to an exception. What would be an appropriate summary for this API?

Reply:
The current summary is the correct summary.  It does nothing, or throws.  It also matches the summary already on IPermission.Demand (which this explicitly implements), which makes it automatically correct :smile:.

https://docs.microsoft.com/en-us/dotnet/api/system.security.ipermission.demand?view=dotnet-plat-ext-3.1

Using `<exception>` makes sense in addition to the summary for consistency, but there's not really any better summary description.

Reply:
I decided to get this merged. After I collect the undoc API report, I'll let you know which ones are missing, which will include this one.